tests/scripts/test_public_api_report.py

```diff
@@ -20,7 +20,7 @@
     parse_docs_contents_for_class_names,
 )
 
-pytestmark = pytest.mark.docs
+pytestmark = pytest.mark.filesystem
```

Comment:
why is this changing

Reply:
This was the only other `docs` marker in our codebase, and it can technically be considered a `filesystem` test. I just wanted to run the `docs-integration` test as part of a separate step